src/frontend/Reply.tsx

```diff
@@ -46,6 +51,7 @@ export const Reply: React.FunctionComponent<RouteComponentProps<{ letterUuid: st
               status,
               letterUuid: letter.uuid,
               content: replyContent.value,
+              statusTimestamp: new Date(),
```

Comment:
I double check with the `updateReply` function, and it is safe to assume that every time this function is called, it will always update the reply status which means that the timestamp should also be updated (automatically). Then, I think the model would be the good place to update it.

Reply:
That is a good check you did, I did not even think about that possibility 👍